### ISSUE TITLE
Impossible to compile against libmemcached without c++11 with GCC 4.9

### ISSUE THREAD

**Issue by m6w6** [awesomized/libmemcached#44]
* *Reported as:* [lp:1328985](https://bugs.launchpad.net/bugs/1328985)
 * *Reported by:* [lp:~adevress](https://launchpad.net/~adevress)
 * *Reported at:* 2014-06-11T17:10:58Z

Imported from Launchpad using lp2gh.

------------------------------------

libmemcached is broken in Fedora rawhide and with any compilation implying GCC 4.9.

The header memcache.h exposes "#include <cinttypes>" which forces the usage of C++11 r triggers compilation error

https://bugzilla.redhat.com/show_bug.cgi?id=1106145
http://koji.fedoraproject.org/koji/taskinfo?taskID=6951772

https://kojipkgs.fedoraproject.org//work/tasks/7102/6957102/build.log

In file included from /usr/include/c++/4.9.0/cinttypes:35:0,
                 from /usr/include/libmemcached-1.0/memcached.h:46,
                 from /usr/include/libmemcached/memcached.h:39,
                 from /builddir/build/BUILD/dmlite-plugins-memcache-0.6.2/src/Memcache.h:7,
                 from /builddir/build/BUILD/dmlite-plugins-memcache-0.6.2/src/Memcache.cpp:11:
/usr/include/c++/4.9.0/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \

This is an urgent fix and block compilation existing software compiling against libmemcached

**Comment by m6w6** [comment]
* *Comment by:* [lp:~adevress](https://launchpad.net/~adevress):
 * *Created at:* 2014-06-11T17:11:43Z

-----
A solution to this problem is to replace :

```
#include <cinttypes>
```

par 
```
#  if ( __cplusplus > 201103L )
#    include <cinttypes>
#  else
#    include <tr1/cinttypes>
#  endif
```

**Comment by m6w6** [comment]
* *Comment by:* [lp:~adevress](https://launchpad.net/~adevress):
 * *Created at:* 2014-06-13T07:53:27Z

-----
Any news on this ? Do you want a patch ?

**Comment by m6w6** [comment]
* *Comment by:* [lp:~adevress](https://launchpad.net/~adevress):
 * *Created at:* 2014-06-25T09:38:58Z

-----
Guys ?

Still no news on this ?

**Comment by m6w6** [comment]
* *Comment by:* [lp:~adevress](https://launchpad.net/~adevress):
 * *Created at:* 2014-08-22T12:37:45Z

-----
Any News on this ?

**Comment by m6w6** [comment]
Fixed in 701592b9754d2ace633882791c755826e3a8af68
